docs/zh-Hans/Tutorials/Part-1.md

```diff
@@ -180,12 +185,12 @@ namespace Acme.BookStore.EntityFrameworkCore
 }
 ````
 
-* `BookStoreConsts` 含有用于表的架构和表前缀的常量值. 你不必使用它,但建议在单点控制表前缀.
-* `ConfigureByConvention()` 方法优雅的配置/映射继承的属性,应始终对你所有的实体使用它.
+* `BookStoreConsts` 含有用于表的架构和表前缀的常量值. 使用它不是强制的,但建议在统一的地方控制表前缀.
+* `ConfigureByConvention()` 方法优雅的配置/映射继承的属性,应对所有的实体使用它.
 
 ### 添加数据迁移
 
-启动模板使用[EF Core Code First Migrations](https://docs.microsoft.com/zh-cn/ef/core/managing-schemas/migrations/)创建和维护数据库架构. 我们应该创建一个新的迁移并且应用到数据库.
+本示例使用[EF Core Code First Migrations](https://docs.microsoft.com/zh-cn/ef/core/managing-schemas/migrations/).因为我们修改了数据库映射配置，我们必须创建一个新的迁移并且应用到数据库.
 
 在 `Acme.BookStore.EntityFrameworkCore.DbMigrations` 目录打开命令行终端输入以下命令:
```

Comment:
```suggestion
在 `Acme.BookStore.EntityFrameworkCore` 目录打开命令行终端输入以下命令:
```
the `Acme.BookStore.EntityFrameworkCore.DbMigrations` has been removed for latest template